src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Matcher/OrderMatcher.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace ScottLogic.Internal.Training.Matcher
+{
+    public class OrderMatcher
+    {
+        public Trade CurrentTrade { get; set; }
+        public List<Order> ExistingOrders { get; set; } = new List<Order>();
+
+        public OrderMatcher()
+        {
+            CurrentTrade = null;
```

Comment:
We have 2 different ways to initialise the properties of this class. Both ways are fine, but I would pick one and use it for everything.  What you have on line 9 is the new, more modern way, so I would probably do that. Initialising in the constructor is the old way, but also fine. Note that this only applies to simple inits, if you need to assign something that is passed in then obviously that needs to be in the constructor.

Reply:
ok, so an uninitialized member variable is null by default, I don't need to say that it is null in the constructor